### PR TITLE
Add make console to Makefile

### PR DESCRIPTION
**Pull request by felixtheflex**
### Context

Add the propensity to ssh console to application via the `make` command

### Changes proposed in this pull request

Add the propensity to ssh console to application via the `make` command

### Checklist

- [ ] Rebased master
- [ ] Cleaned commit history
- [ ] Tested by running locally

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -111,3 +115,6 @@ edit-app-secrets: install-fetch-config ## Edit Find App Secrets
 print-app-secrets: install-fetch-config ## View Find App Secrets
 	. terraform/workspace_variables/$(DEPLOY_ENV).sh && bin/fetch_config.rb -s azure-key-vault-secret:$${TF_VAR_key_vault_name}/$${TF_VAR_key_vault_app_secret_name} \
 		-f yaml
+console:
+	cf target -s ${space}
+	cf ssh find-${DEPLOY_ENV} -t -c "cd /app && /usr/local/bin/bundle exec rails c"
```
Comment (vigneshmsft) [inline Makefile:120]: find's production app name is `find-prod`

File: Makefile [hunk 2]
```diff
@@ -111,3 +115,6 @@ edit-app-secrets: install-fetch-config ## Edit Find App Secrets
 print-app-secrets: install-fetch-config ## View Find App Secrets
 	. terraform/workspace_variables/$(DEPLOY_ENV).sh && bin/fetch_config.rb -s azure-key-vault-secret:$${TF_VAR_key_vault_name}/$${TF_VAR_key_vault_app_secret_name} \
 		-f yaml
+console:
```
Comment (vigneshmsft) [inline Makefile:118]:
```suggestion
.PHONY: console ## start a rails console, eg: make qa console
console:
```

**Review by vigneshmsft — COMMENTED** [review]
nice one

File: Makefile (hunk 2 above)
Comment (felixtheflex) [reply Makefile:124]: done..

**Review by vigneshmsft — APPROVED** [review]

**Pull request merged**